.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: Report a bug or unexpected behavior.
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please add as much relevant information to the form below as you can.
+        This will help us replicate and solve the issue. Thank you!
+  - type: dropdown
+    id: backend
+    attributes:
+      label: Backend
+      description: What backend of gluten are you running?
+      options:
+        - VL (Velox)
+        - CH (ClickHouse)
+    validations:
+      required: true
+      
+  - type: textarea
+    id: description
+    attributes:
+      label: Bug description
+      description: Please describe the issue and the expected behavior.
+      value: "[Expected behavior] and [actual behavior]."
+    validations:
+      required: true
+  - type: textarea
+    id: info
+    attributes:
+      label: System information
+      description: Please run `scripts/info.sh` and paste the output here.
+    validations:
+      required: true
```

Comment:
Can we allow the missing of this part in bug report? For some bug, it may be obviously not related to system environment. So I think `required: false` may be better.